{"name": "test_excel[c-gap-inner-right-zero{padding-right:0}\n[data-pmd] .c-gap-inner-bottom-zero{padding-bottom:0}\n[data-pmd] .c-gap-inner-left-zer]", "status": "failed", "statusDetails": {"message": "AssertionError: assert b'c-gap-inner...nner-left-zer' == '<Response [200]>'\n  +b'c-gap-inner-right-zero{padding-right:0}\\\\n[data-pmd] .c-gap-inner-bottom-zero{padding-bottom:0}\\\\n[data-pmd] .c-gap-inner-left-zer'\n  -'<Response [200]>'", "trace": "data = b'c-gap-inner-right-zero{padding-right:0}\\\\n[data-pmd] .c-gap-inner-bottom-zero{padding-bottom:0}\\\\n[data-pmd] .c-gap-inner-left-zer'\n\n    @pytest.mark.parametrize('data', ExcelHandler.get_excel_data(1))\n    def test_excel(data):\n>       assert data == '<Response [200]>'\nE       AssertionError: assert b'c-gap-inner...nner-left-zer' == '<Response [200]>'\nE         +b'c-gap-inner-right-zero{padding-right:0}\\\\n[data-pmd] .c-gap-inner-bottom-zero{padding-bottom:0}\\\\n[data-pmd] .c-gap-inner-left-zer'\nE         -'<Response [200]>'\n\ntest.py:16: AssertionError"}, "parameters": [{"name": "data", "value": "<class 'bytes'>"}], "start": 1586663330718, "stop": 1586663330718, "uuid": "2097f08c-b02c-47a3-9682-feb4fb111c76", "historyId": "ea5f59e64789818d64317e8f1e53c5f5", "testCaseId": "52e4857383463840bb3d3e8e44c10486", "fullName": "allure.test#test_excel[c-gap-inner-right-zero{padding-right:0}\n[data-pmd] .c-gap-inner-bottom-zero{padding-bottom:0}\n", "labels": [{"name": "parentSuite", "value": "allure"}, {"name": "suite", "value": "test"}, {"name": "host", "value": "localhost"}, {"name": "thread", "value": "42329-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "allure.test"}]}